### ISSUE TITLE
Python and Flask Tutorial | Optional Activities Step 10 | Set ENV variable

### ISSUE THREAD

**Issue by kimfucious** [Microsoft/vscode-docs#1783]
After following the steps under Optional Activities, [here](https://code.visualstudio.com/docs/python/tutorial-flask), the app will run in the debugger; however, the app does not (at least for me) run using `python3 -m flask run`.  The command returns the following error:

`Error: Could not locate a Flask application. You did not provide the "FLASK_APP" environment variable, and a "wsgi.py" or "app.py" module was not found in the current directory.`

I assume that this works in debug, because (step 7) we modified the env in launch.json to hello_app.webapp:

```
      "name": "Python: Flask (0.11.x or later)",
      "type": "python",
      "request": "launch",
      "module": "flask",
      "env": {
        "FLASK_APP": "hello_app.webapp"
      },
      "args": ["run", "--no-debugger", "--no-reload"]
```
While there is a tip at the end of **Create and run a minimal Flask app** that lightly touches upon environment variables, there probably should be another, more detailed explanation of this in the Optional Activities section to instruct the person following this tutorial on how any why this is needed.

Perhaps adding the following could help to make the tutorial more clear:

11.  Change into the hello_app directory `cd hello_app`
12. Add an environment variable for webapp.py, as Flask needs to know we changed away from the default of app.py.  `export set FLASK_APP=webapp`
13. Run python3 -m flask run

**Comment by bernard-murunga** [comment]
@kimfucious Thanks a lot. I've been trying to find a solution for hours. I really appreciate the answer.

**Comment by kraigb** [comment]
@kimfucious Thanks also for your input.  I'll update the tutorial in early September when I'm back from an extended holiday!

**Comment by kraigb** [comment]
@kimfucious Take a look at https://github.com/Microsoft/vscode-docs/commit/edabaf38cf90bb24ad42295d96c9a17d65d09ab8 and let me know if that change works for you.

**Comment by kraigb** [comment]
@kimfucious Yes, we enjoyed our vacation (almost three weeks in England). Thanks for the quick review--the changes will be merged in with others for the next release of the extension. In the meantime, I'm closing the issue here.

**Comment by muckisg** [comment]
@kimfucious Working in powershell the export of enviroment varaiables seems to be a bit different (sorry, no general windows user):
`$Env:FLASK_APP=webapp`
Control the entry with 
`Get-ChildItem Env:FLASK_APP`

**Comment by developerzzz** [comment]
for windows user please use the following
1) activate python environment 
2) navigate to the root folder of application
3) run the following command $env:FLASK_APP ="yourapp.py"
4) flask run

**Comment by kraigb** [comment]
@developerzzz Can you open a new issue with your previous comment? The docs team generally doesn't follow comments to already-closed issues like this one, but will see new issues.

**Comment by syeda0521** [comment]
Hi, I'm new to flask. I am a Windows user and on my cmd I'm trying to make flask run after "set FLASK_APP = flaskblog.py" flaskblog.py being my file name. I have already created a virtual environment and I am in my virtual environment. I'm not sure what the problem is. I have attached a snippet as well. 
![error](https://user-images.githubusercontent.com/40681829/84695478-72cc8400-af7d-11ea-9481-2e3f7ec7468d.JPG)
)
